src/i18n/DopplerIntlProvider.js

```diff
@@ -14,7 +14,10 @@ const messages = {
 addLocaleData([...en, ...es]);
 
 export default ({ locale, children }) => (
-  <IntlProvider locale={locale || 'en'} messages={flattenMessages(messages[locale || 'en'])}>
+  <IntlProvider
+    locale={locale || 'en'}
+    messages={flattenMessages(messages[locale || 'en'] || messages['en'])}
```

Comment:
only format change?

Reply:
No, now when `messages[locale || 'en']` is _falsy_ we will use `messages['en']`